Add a getting started guide (#457)
viamrobotics/viam-python-sdk: SETUP.md

# Viam + Python

This guide will help you get set up with Viam with Python. It assumes that you are starting from scratch, and will walk you through setting up a fresh environment and installing the necessary requirements.

## Setup your project

The first step is to create a directory to house your project. For this guide, we will be using the directory name `viam-python`:

```bash
mkdir viam-python
cd viam-python
```

## Create a Virtual Environment

Now that we are in the project directory, let's create and activate a virtual environment for python to run in.

> **INFO**

> > Creating a virtual environment (`venv`) is important as it isolates this python environment from any other you might already have. This allows us to ensure a clean project and easier dependency management, as well so not bloating your global python environment.

Reviewer: ```suggestion
> Creating a virtual environment (`venv`) is important as it isolates this python environment from any other you might already have. This allows us to ensure a clean project and easier dependency management, and it avoids bloating your global python environment.
```

```bask

> python3 -m venv viam-env

Reviewer: Does venv come by default with pip3?

Author: I tested it on various docker instances and it turns out that ubuntu doesn't come with the necessary requirements (at least not when I run it in docker). So I added a note below to guide users

source viam-env/bin/activate
```

You will now see `(viam-env)` prepend the commands in your terminal window. This shows that the python packages being used are from this particular environment.

You can exit this environment by running `deactivate`.

## Install Viam

Inside the activated `viam-env` python environment, you can now install the Viam SDK:

```bash

> pip3 install viam-sdk

Reviewer 2: Should we also add a note about the numpy optional dependency?

Author: Good call, added a note below
